### DISCUSSION TITLE
Custom colours not being included in CSS

### DISCUSSION THREAD

**Discussion by individyouall** [tailpress/tailpress#103] [General]
Hi
Thanks for a great theme.
One question I have though, relates to custom colours using the theme.js. I have set up a colour with 'light-grey' slug and then using an ACF block I then pull in the Gutenberg background colour. This then creates a classname that is being applied that should work w/ Tailwind (bg-light-grey). The problem is that no matter if I run a dev or production build, the .bg-light-grey class is not being added to the app.css file.
Any ideas how I can get this to work? It works with .bg-primary for reference.
Thanks
Adam
<img width="702" alt="Screenshot 2022-04-06 at 17 36 23" src="https://user-images.githubusercontent.com/103135568/162024750-444421b4-1840-492d-91db-3c465935f54c.png">
<img width="575" alt="Screenshot 2022-04-06 at 17 37 01" src="https://user-images.githubusercontent.com/103135568/162024766-95d2c8d6-6621-4b96-b95f-2481dc6f8556.png">
<img width="486" alt="Screenshot 2022-04-06 at 17 37 21" src="https://user-images.githubusercontent.com/103135568/162024775-481ade4b-ca2a-4ee7-827c-60f828105cc6.png">
<img width="324" alt="Screenshot 2022-04-06 at 17 37 45" src="https://user-images.githubusercontent.com/103135568/162024782-7adfe5bc-cc1e-48fd-9ba2-e495717ea984.png">
<img width="375" alt="Screenshot 2022-04-06 at 17 38 49" src="https://user-images.githubusercontent.com/103135568/162024789-17068005-435f-4907-ae76-b7a77daafdcc.png">

**Comment by jeffreyvr** [comment]
Hi, are you watching the files in which you use those classes? The files you are using the Tailwind classes in should be added here:

https://github.com/jeffreyvr/tailpress/blob/master/tailwind.config.js#L6

Otherwise they are not noticed and will be purged.

**Comment by individyouall** [comment]
Thank you for the reply. That wasn't the solution but it has led me to one. The problem is that I was applying the css class via Gutenberg, so the php files didn't include the actual classname. As I result, I think I'll just have to manually add this into the custom.css file like the attached

<img width="332" alt="Screenshot 2022-04-07 at 10 24 02" src="https://user-images.githubusercontent.com/103135568/162167078-c5d6efd1-121e-4c5f-8999-116c1aafa618.png">
.

**Reply by individyouall** [reply]
Apply doesn't actually work, so for others, I had to do it like this instead:

<img width="484" alt="Screenshot 2022-04-07 at 10 33 24" src="https://user-images.githubusercontent.com/103135568/162168956-243eeb72-169d-4e5c-a43a-b53a20aaf7a7.png">
